plugin.video.mvmix/resources/lib/artist_player.py

```diff
@@ -0,0 +1,103 @@
+# -*- coding: utf-8 -*-
+
+import random
+import sys
+import xbmc
+import xbmcgui
+import xbmcplugin
+
+class mvmixArtistPlayer(xbmc.Player):
```

Comment:
Plugins should not use xbmc.Player this way. Why don't you just create a playlist?

Reply:
Sorry didnt know. I use it to create a playlist to automatically add new videos everytime a new video starts.
As this is core functionality, feel free to do whatever with this pr. 
Im not able to rethink/rework this anytime soon.